ci-operator/step-registry/cucushift/installer/check/realtime/cucushift-installer-check-realtime-commands.sh

```diff
@@ -37,5 +37,19 @@ else
     exit 1
 fi
 
+rets=$(oc get machineconfigs | grep realtime-master)
```

Comment:
@sgaoshang I just had a check on our official documentation, seems it's only clarified the RT kernel only supported for workers.
https://docs.openshift.com/container-platform/4.14/post_installation_configuration/machine-configuration-tasks.html#nodes-nodes-rtkernel-arguments_post-install-machine-configuration-tasks
`Currently, real-time kernel is supported only on worker nodes `
So I guess don't need enabling it on master and check on masters.

Reply:
For GCP RT kernel workflow it's only configured for workers - https://steps.ci.openshift.org/reference/ipi-conf-gcp#line68

Reply:
I checked that there is no real time kernel cases on vsphere.
Agree with Gaoyun to only configure RT kernel for workers as it is described in official doc.

Reply:
Fixed, thanks.